code/modules/power/lighting.dm

```diff
@@ -514,14 +514,12 @@
 	status = LIGHT_EMPTY
 	update()
 
-/obj/machinery/light/proc/broken(skip_sound_and_sparks = 0)
-	if(status == LIGHT_EMPTY || status == LIGHT_BROKEN)
+/obj/machinery/light/proc/broken(skip_sounds_and_sparks = 0)
+	if(status == LIGHT_EMPTY || status == LIGHT_BROKEN || status == LIGHT_BURNED)

```

Comment:
Err, you can still break a burnt out lightbulb; I don't think because a light is burnt out it should be immune to someone whacking it.